sky/task.py

```diff
@@ -423,7 +424,7 @@ def from_yaml(yaml_path: str) -> 'Task':
           ValueError: if the path gets loaded into a str instead of a dict; or
             if there are any other parsing errors.
         """
-        with open(os.path.expanduser(yaml_path), 'r') as f:
+        with open(os.path.expanduser(yaml_path), 'r', encoding='utf-8') as f:
```

Comment:
Why change encoding here?

Reply:
It is recommended to always specify the encoding in `open` function. We did not change the encoding, but just use the default `utf-8`. https://peps.python.org/pep-0597/